app/controllers/concerns/idv/document_capture_concern.rb

```diff
@@ -58,7 +58,7 @@ def selfie_requirement_met?
         stored_result.selfie_check_performed?
     end
 
-    def redirect_to_correct_vendor(vendor, in_hybrid_mobile)
+    def redirect_to_correct_vendor(vendor, in_hybrid_mobile: false)
```

Comment:
I kinda feel like `in_hybrid_mobile` shouldn't have a default value.

Reply:
Removed in 19dce93a14